adot/python/src/template.yml

```diff
@@ -0,0 +1,52 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: OpenTelemetry Python Lambda layer for Python 3.8 Runtime
+Parameters:
+  LayerName:
+    Type: String
+    Description: Lambda layer name to be published
+    Default: adot-opentelemetry-python-38
+Resources:
+  OTelLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: !Ref LayerName
+      Description: Opentelemetry Python38 layer
+      ContentUri: ./otel
+      CompatibleRuntimes:
+        - python3.8
+    Metadata:
+      BuildMethod: makefile
+  api:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: api
+      TracingEnabled: true
+      OpenApiVersion: 3.0.2
+  function:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: lambda_function.lambda_handler
+      Runtime: python3.8
+      CodeUri: ./function
+      Description: Build ADOT Python Lambda layer and sample app from scratch
+      MemorySize: 512
+      Timeout: 15
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSLambda_ReadOnlyAccess
+        - AWSXrayWriteOnlyAccess
+        - AmazonS3FullAccess
```

Comment:
Only give S3ListBuckets

Reply:
AWSXrayWriteOnlyAccess is also needed

Reply:
I meant don't give full access, it needs to be something smaller (at least ReadOnly if not the fine grained permission)